readme.md

```diff
@@ -54,6 +58,10 @@ Then you will need to run this to install the ruby dependencies.
 
     bundle install
 
+Or for short just
```

Comment:
I don't think we need to include the short hand in our docs. 🤔

Reply:
I can remove it. I felt it does not hurt to go a bit into detail though.